pleskdistup/common/src/dns.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023-2024. WebPros International GmbH. All rights reserved.
+import os
+from collections import deque
+
+
+def get_all_includes_from_bind_config(config_file: str) -> list:
```

Comment:
`list[str]`

Reply:
OK.
BTW, could you remind me the minimal Python version we're targeting?

Reply:
3.6